mantidproject/download.mantidproject.org
# Split up and overhaul generation scripts

- Chunk major steps into separate componenets
- Introduce data classes instead of lists
- Rejig loops / os.path usage to pathlib
- Add tests with pytest
- Update Jinja to understand new data structs
- Drop Paraview support / mentions (these were only lists for 3.3)
- Add venv instructions

**To test**
- Follow instructions on README
- Check the generated pages match our [existing ones](https://download.mantidproject.org)
